### PR TITLE
add possibility to remove VEvent from calendar by id

### PR DESCRIPTION
**Pull request by zachmann**
This adds a simple method to remove an event from a calendar. This addresses #57

### CODE REVIEW

**Comment by arran4** [comment]
LGTM. Running workflow.

**Review by arran4 — APPROVED** [review]

**Comment by arran4** [comment]
Any change of a test before I merge?

File: calendar.go [hunk 1]
```diff
@@ -425,6 +425,23 @@ func (calendar *Calendar) AddVEvent(e *VEvent) {
...
+				if len(calendar.Components) > i+1 {
+					calendar.Components = append(calendar.Components[:i], calendar.Components[i+1:]...)
+				} else {
+					calendar.Components = calendar.Components[:i]
+				}
+				return
+			}
+		}
+	}
+	return
```
Comment (arran4) [inline calendar.go:442]: @zachmann Oh lint "error."

**Comment by zachmann** [comment]
> Any change of a test before I merge?

I looked for a test for `AddEvent` which is could use as a base line for a `RemoveEvent` test, but there are no tests for adding an events, so I left out the test.

**Comment by arran4** [comment]
All good.

**Comment by arran4** [comment]
```
 * [new tag]         v0.2.1 -> v0.2.1
```

**Pull request merged**